Tests/TestUtilities/TestUtils.cs

```diff
@@ -234,7 +234,7 @@ public static string CreateFile(string parentDir, string fileName, string conten
         /// Creates a new project info file in a new subdirectory with the given additional properties.
         /// </summary>
         public static string CreateProjectInfoInSubDir(string parentDir,
-            string projectName, Guid projectGuid, ProjectType projectType, bool isExcluded, string fullProjectPath, string encoding,
+            string projectName, string projectLanguage, Guid projectGuid, ProjectType projectType, bool isExcluded, string fullProjectPath, string encoding,
```

Comment:
As the argument is provided only once and every other time is is null,  it could be `string projectLanguage = null`, not to repeat it all the time.

Reply:
I wanted to enforce every new or refactored test using this to consider C#/VB datarows.